addons/mail/models/mail_activity.py

```diff
@@ -861,3 +861,70 @@ def activity_unlink(self, act_type_xmlids, user_id=None):
             domain = ['&'] + domain + [('user_id', '=', user_id)]
         self.env['mail.activity'].search(domain).unlink()
         return True
+
+    def _read_progress_bar(self, domain, group_by, progress_bar):
```

Comment:
We have a section "ORM overrides" in the file, could be great to order code as already done. It allows to quickly understand file organization and avoid having randomly placed code. Especially this one that is very orm oriented and low level.